weave-js/src/components/PagePanelComponents/Home/Browse3/charts/ChartModal.tsx

```diff
@@ -404,7 +405,51 @@ export const ChartModal: React.FC<ChartModalProps> = ({
               p: 3,
               backgroundColor: MOON_50,
             }}>
-            <SectionHeader first>Plot Type</SectionHeader>
+            <SectionHeader first>Chart Name</SectionHeader>
+            <TextField
+              value={localConfig.customName || ''}
+              onChange={e =>
+                setLocalConfig(prev => ({
+                  ...prev,
+                  customName: e.target.value || undefined,
+                }))
+              }
+              placeholder={
+                localConfig.yAxis &&
+                localConfig.plotType &&
+                localConfig.aggregation
+                  ? generateChartAutoName(
+                      localConfig.yAxis,
+                      localConfig.plotType,
+                      localConfig.aggregation,
+                      localConfig.xAxis || 'started_at',
+                      callData
+                    )
+                  : 'Chart name'
+              }
+              variant="outlined"
+              size="small"
+              fullWidth
+              sx={{
+                mb: 2,
+                '& .MuiInputBase-input': {
+                  fontSize: '14px',
+                  fontFamily: 'inherit',
+                },
+                '& .MuiInputBase-input::placeholder': {
+                  color: '#999',
+                  opacity: 1,
+                  fontSize: '14px',
+                  fontFamily: 'inherit',
+                },
+                '& .MuiOutlinedInput-root': {
+                  fontSize: '14px',
+                  fontFamily: 'inherit',
+                },
+              }}
```

Comment:
Tailwind please?

Reply:
There is a mui / tailwind integration https://mui.com/material-ui/integrations/tailwindcss/tailwindcss-v4/ but I don't think we have it set up and this styling pattern is being used for many of our components elsewhere. But in this case there is no reason for me to self roll an input form here so I updated this to just use a canned component from our library